Common/Source/Dialogs/dlgTerminal.cpp

```diff
@@ -261,6 +262,12 @@ void dlgTerminal(int portnumber) {
   OnPortClicked(NULL);
 
 
+  if(wf)
+  {
+	WndProperty* wp = (WndProperty*)wf->FindByName(TEXT("prpSendText"));
+    wp->SetText(TxText);
+    wp->GetDataField()->Set(TxText);
+  }
```

Comment:
WndProperty::SetText must be used only if  WndProperty::GetDataField() return nullptr
instead use : 
``` cpp 
wp->GetDataField()->Set(TxText);
wp->RefreshDisplay();
```